tests/test_project/test_project/viewtest/tests.py

```diff
@@ -118,6 +119,23 @@ def test_materialized_view(self):
         self.assertEqual(models.MaterializedRelatedViewWithIndex.objects.count(), 1,
             'Materialized view should have updated concurrently')
 
+    def test_signals(self):
+        synced_views = []
+        all_views_were_synced = [False]
+
+        @receiver(view_synced)
+        def on_view_synced(sender, has_changed, **kwargs):
+            synced_views.append(sender)
```

Comment:
I've changed this test to also verify signal arguments for two models (one for a view and one for a materialized view) - is that what you had in mind? The test is starting to get a bit verbose, but now we know exactly what is getting sent.